pkg/conversion/queryparams/convert.go

```diff
@@ -112,6 +127,8 @@ func Convert(obj runtime.Object) (url.Values, error) {
 			if isValueKind(ft.Elem().Kind()) {
 				addListOfParams(result, tag, omitempty, field)
 			}
+		case isPointerKind(ft.Kind()):
+			addPointerParam(result, tag, omitempty, field)
```

Comment:
It'd be good to check whether the pointer is to one of the supported types above (value or array/slice). Maybe the check should be done before the switch and then use ft.Elem().Kind() (in case of pointer) as the kind to check.